### ISSUE TITLE
The systray icon is a little bit dim

### ISSUE THREAD

**Issue by matthewsheeran** [m0ngr31/VirtualDesktopManager#22]
You probably want to thicken the lines a little e.g. by doubling them.
Here it is in comparison with a range of both native and add on systray icons/programs like PortableApps, Outlook, VMware, Minibin and that '1' is barely even visible:
![capture](https://user-images.githubusercontent.com/2776826/29749083-bfe2b6ca-8b67-11e7-863a-956a83fb8bd8.PNG)
Compare with other text like the clock time and date.
Also if Explorer crashes or is restarted the icon often disappears without getting recreated and  VirtualDesktopManager often also crashes.

**Comment by matthewsheeran** [comment]
Funny how it appears to look better on my new Laptop's 4K Display even though it is scaled back down to 1920x1080 when output to an external Monitor at that resolution (not the internal 4K scaled back to 1920x1080 which looks as gorgeous as full scale 4K even at lower resolutions):
![capture](https://user-images.githubusercontent.com/2776826/29856159-610e67c8-8d94-11e7-9c60-ad74d76a5a1d.PNG)
Go figure! Brownie points for anyone who can explain why better than I can!?:
I am a GIS guy who knows rasters and both machines have the same plain black wallpaper and taskbar defaults but the new machine's display looks a damn sight better at that same monitor same resolution so can only be an ever so slight difference in Taskbar scaling (which indeed there is and probably shifts the pixels from say 50% brightness across 2 closer to say 100% across 1 as a consequence) and/or else the quality of the newer Intel Graphics Chipset output makes the difference.
So I guess I can now live with it on my new machine!

**Comment by sunnyape** [comment]
I agree. The icon itself is a bit small and uninformative. A simpler rectangle that looked like a screen with a larger numeral inside might be better. The 'arrow' part of the icon isn't really adding anything value. Maybe something like this:

![capture](https://user-images.githubusercontent.com/32824881/31590918-c0dd3298-b264-11e7-984d-39849a6bffd4.PNG)

**Comment by matthewsheeran** [comment]
Yeah sounds good. The original icon varies and is sometimes faded out
though I am having better luck now with a newer HDMI cable there will still
be poor appearance for those using lower resolutions.
Regards
Matthew

On 16 Oct. 2017 11:26 am, "sunnyape" <notifications@github.com> wrote:

I agree. The icon itself is a bit small and uninformative. A simpler
rectangle that looked like a screen with a larger numeral inside might be
better. The 'arrow' part of the icon isn't really adding anything value.
Maybe something like this:

[image: capture]
<https://user-images.githubusercontent.com/32824881/31590918-c0dd3298-b264-11e7-984d-39849a6bffd4.PNG>

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/m0ngr31/VirtualDesktopManager/issues/22#issuecomment-336753274>,
or mute the thread

**Comment by sunnyape** [comment]
I've found another nice app to use in conjunction with VDM to help you know what desktop you're currently on. It's called TranslucentTB (https://github.com/TranslucentTB). If you use VDM to change wallpapers and you use TranslucentTB to make the taskbar totally transparent or blurred, you get much greater visual feedback as to what desktop you're on.
